.circleci/config.yml

```diff
@@ -175,6 +167,42 @@ jobs:
             cd generated_docs/
             mkdocs gh-deploy \
               -m "[ci skip] Deployed {sha} with MkDocs version: {version}"
+  generate-sql:
+    docker: *docker
+    steps:
+      - checkout
+      - *skip_forked_pr
```

Comment:
Would this ever be run from a forked PR?

Reply:
Good point. Will remove.